infra/growth/src/resources/authentication.js

```diff
@@ -201,10 +201,11 @@ async function createInviteCode(accountId) {
    * IMPORTANT: Frontend expects 11 letter invite code. Be mindful of that
    * if changing invite code creation logic
    */
-  const code =
+  const code = (
     `${accountId.substring(2, 5)}` +
     `${accountId.substring(accountId.length - 3, accountId.length)}` +
     `${Math.round(Math.random() * 1000000).toString(16)}`
+  ).padEnd('1')
```

Comment:
Where does the Front-End check for 11 characters invite code?

The logic above may generate codes that are anywhere between 7 and 11 characters, depending on the value returned by Math.random().

For example, if Math.random() return 0.000001, Math.round(Math.random() * 1000000).toString(16) returns '1' and the code will be 7 chars long.

The prod database reflects this:
```
import=# select length(code), count(*) from growth_invite_code group by length(code);
 length | count  
--------+--------
      7 |      1
      8 |     26
      9 |    413
     10 |   6818
     11 | 103391
```

So I don't think we need to add the padEnd

Reply:
On master we still have this check, have removed it in this branch. See Welcome.js page changes in this PR. And yes agreed, with changes in Welcome page this change is no longer needed. Removing it....